C/jets.c

```diff
@@ -74,6 +107,15 @@ AND_(16)
 AND_(32)
 AND_(64)
 
+/* or_1 : TWO * TWO |- TWO */
+bool or_1(frameItem* dst, frameItem src, const txEnv* env) {
+  (void) env; /* env is unused. */
+  bool x = readBit(&src);
+  bool y = readBit(&src);
+  writeBit(dst, x || y);
```

Comment:
In theory the boolean operators are short-circuited.  Of course, there are no side-effects at play here, and the compiler is free to do whatever it thinks is best.

Reply:
The other difference is that the boolean operators are guaranteed to return `0` or `1`, but IIUC this does not make a difference here because `readBit` also offers this guarantee.

My feeling is that the code is slightly easier to read with the bitwise operators because they're also used in the macros. Using the same operators saves the reader from thinking about why different operators are used.  But it's ultimately a question of personal preference, and both approaches are correct, of course.